app/src/main/res/layout/activity_splash.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.activities.splash"
+    android:orientation="vertical"
+    android:gravity="center"
+    android:background="#ffff">
+
+    <ImageView
+        android:layout_width="250dp"
```

Comment:
I would suggest you to not use the hard coded values as they might give results that are device specific